database/src/mdbx/database.rs

```diff
@@ -10,8 +10,8 @@ use crate::{
     Error,
 };
 
-const GIGABYTE: usize = 1024 * 1024 * 1024;
-const TERABYTE: usize = GIGABYTE * 1024;
+pub const GIGABYTE: usize = 1024 * 1024 * 1024;
+pub const TERABYTE: usize = GIGABYTE * 1024;
```

Comment:
Hmm, sorry, it seems like not all of my comments made it through. I wanted to comment here that I don't think it's good for the database crate to export `GIGABYTE`/`TERABYTE` constants as that seems pretty unrelated to databases.

```suggestion
const GIGABYTE: usize = 1024 * 1024 * 1024;
const TERABYTE: usize = GIGABYTE * 1024;
```